Documentation/design/operatorgroups.md

```diff
@@ -0,0 +1,221 @@
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: my-group
+  namespace: my-namespace
+  targetNamespaces:
+  - my-namespace
+  - my-other-namespace
+  - my-other-other-namespace
+```
+
+> Note: If both `spec.targetNamespaces` and `spec.selector` are defined, `spec.selector` is ignored.
+
+Additionally, a _global_ `OperatorGroup` (which selects all namespaces) is specified by omitting both `spec.selector` and `spec.targetNamespaces`:
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: my-group
+  namespace: my-namespace
+```
+
+The resolved set of selected namespaces is surfaced in an `OperatorGroup`'s `status.namespaces` field. A global `OperatorGroup`'s `status.namespace` is of length 1 and contains the empty string, `""`, which signals a consuming operator that it should watch all namespaces.
+
+> Note: The consuming operator must know to treat `""` as an all namespace configuration.
+
+## OperatorGroup CSV Annotations
+
+Member CSVs of an `OperatorGroup` get the following annotations:
+* `olm.operatorGroup=<group-name>`
+  * Contains the name of the `OperatorGroup`
+* `olm.operatorGroupNamespace=<group-namespace>`
+  * Contains the namespace of the `OperatorGroup`
+* `olm.targetNamespaces=<target-namespaces>`
+  * Contains a comma-delimited string listing the `OperatorGroup`'s target namespace selection. This annotation is projected onto the pod template of a CSV's deployments and can be consumed by a pod instance via [The Downward API](https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#the-downward-api)
+* If the CSV does not [have a Copied status reason](#copied-csvs)
+
+> Note: All annotations except `olm.targetNamespaces` are included with [copied CSVs](#copied-csvs). 
+
+## Provided APIs Annotation
+
+Information about what GVKs are provided by an`OperatorGroup`are surfaced in an `olm.providedAPIs` annotation. The annotation's value is a string consisting of a set of `<Kind>.<version>.<group>`s delimited with commas. The GVKs of CRDs and APIServices provided by all active member CSVs of an`OperatorGroup`are included.
+
+Here's an example of an`OperatorGroup`with a single active member CSV providing the PackageManifests resource:
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  annotations:
+    olm.providedAPIs: PackageManifest.v1alpha1.packages.apps.redhat.com
+  creationTimestamp: 2019-02-19T16:18:28Z
+  generation: 2
+  name: olm-operators
+  namespace: local
+  resourceVersion: "784"
+  selfLink: /apis/operators.coreos.com/v1alpha2/namespaces/local/operatorgroups/olm-operators
+  uid: fd8298be-3461-11e9-92f5-080027f6a3cd
+spec:
+  selector: {}
+  serviceAccount:
+    metadata:
+      creationTimestamp: null
+  targetNamespaces:
+  - local
+status:
+  lastUpdated: 2019-02-19T16:18:28Z
+  namespaces:
+  - local
+```
+
```

Comment:
I think we should introduce the Copied status and the static provided apis around here; they’re referenced frequently in the intersection section.

Reply:
It's been updated to reflect your comments - forgot to push this morning.